notebooks/example.ipynb

```diff
@@ -200,6 +199,228 @@
     "%%bash\n",
     "cat ./fixtures/rendered_config.yaml"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea5eff92",
+   "metadata": {},
+   "source": [
+    "## Example 3: Comparing two config files\n",
+    "Let's explore using the `config.compare()` method to compare two config files."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "71d27692",
+   "metadata": {},
+   "source": [
+    "We again need to start by importing the `uwtools.api.config` from the uwtools python package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "7fb40514",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from uwtools.api import config\n",
+    "from uwtools.api.logging import use_uwtools_logger\n",
+    "use_uwtools_logger()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b0103f44",
+   "metadata": {},
+   "source": [
+    "Please review the [config.compare() documentation](https://uwtools.readthedocs.io/en/main/sections/user_guide/api/config.html#uwtools.api.config.compare) for full information on this function's arguments."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b121dc29",
+   "metadata": {},
+   "source": [
+    "For example, let's compare two Fortran namelist files with differences:"
```

Comment:
I'm curious if it would be easier to follow this multi-part example if we somehow emphasize this line, and the one before the comparison of semantically identical files. Maybe bold or larger font? Perhaps just making a third level under the headings used for the examples.

Reply:
Bolded both lines